lib/sort.js

```diff
@@ -36,6 +38,15 @@ module.exports = function sortDocs(comments, options) {
   var fixed = options.toc.filter(function (val) {
     return typeof val === 'object' && val.name;
   }).map(function (val) {
+    if (typeof val.file === 'string') {
+      var filename = path.join(process.cwd(), val.file);
+      try {
+        val.description = fs.readFileSync(filename).toString();
```

Comment:
This is currently sync because this pipeline is currently all sync. I can also move it to the async version if that's preferred but that would mean changing more in this file and the interface to `sort`.